feeder/values.yaml

```diff
@@ -71,7 +71,7 @@ env:
  - name: MYSQL_PASSWORD
    value: password 
  - name: MYSQL_URL
-   value: mysql.explorer.svc.cluster.local 
+   value: mysql.accuknox-agents.svc.cluster.local 
```

Comment:
This change is not needed. This change might create problems since the `values.yaml` might be in use in other cases.